### ISSUE TITLE
Object list item: missing layouts variants 

### ISSUE THREAD

**Issue by daisy-yovcheva** [SAP/fundamental-ngx#3747]
#### Is this a bug, enhancement, or feature request?
Enhancement, or feature request

#### Briefly describe your proposal.

**It’s missing the following list layouts for "Text Positioning" with no avatar:** 

-List with attributes and statuses
-Example with more attributes than statuses
-Example with more statuses than attributes
-List with attributes only
-List with statuses only

Refer to Visual core: 
https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2036048188

- Attribute that gets 100% of the width
![Screen Shot 2020-11-04 at 8 24 25 PM](https://user-images.githubusercontent.com/72403541/98197551-aeb5b500-1f2f-11eb-8e84-1328b17555f9.png)

**Comment by sKudum** [comment]
Hi @puru-hk , @daisy-yovcheva , @manjunathanagaraj  adding these examples will lead to duplication of the same examples (with the only diff avatar not present).

The examples which we have right now are having few with avatar and few without. which are added with the intention to avoid duplication.

Please do suggest this. if you feel, having duplication is acceptable in this case.

**Comment by manjunathanagaraj** [comment]
@daisy-yovcheva , I agree with @sKudum . As long as there is an example for both with and without Avatar, this should be good enough. Let us know if there are any concerns.

**Comment by daisy-yovcheva** [comment]
Hi @manjunathanagaraj and @sKudum ,

Yes. I agree with you, but we definitely need an example that has a "Very long title" - taking two rows.
![Screen Shot 2020-11-10 at 12 18 15 PM](https://user-images.githubusercontent.com/72403541/98728964-d9c25d80-23a2-11eb-908e-c5aa8cd39232.png)

Also, I have concerns about the Visual specs - the width of the component and padding between left and right attributes.
Visual core ref: https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2036048188

@kenykim and @pierrom , Could you please check the specs and confirm if any changes are needed.

**Comment by sKudum** [comment]
Hi @daisy-yovcheva,

1).Example added with long title in PR #4079 
Which will shown as below.

In Desktop:
<img width="1610" alt="2020-12-10_19-46-50" src="https://user-images.githubusercontent.com/53487468/101784544-a994e880-3b21-11eb-94f5-6ff8b47643f2.png">


In small screens:

<img width="306" alt="2020-12-10_19-46-30" src="https://user-images.githubusercontent.com/53487468/101784595-b87b9b00-3b21-11eb-97d2-229ef24d3435.png">



2). Regarding I have concerns about the Visual specs - the width of the component and padding between left and right attributes.:

Concerns are raised to Fiori team.  Kunz, Andreas,georg.christoph,Keane, Gerard has confirmed that. enhancement work on  object list item styles is going on. soon we can expect an updated set of styles for it. Till then we can proceed with whatever has been shared in Fiori visual specs.
